### DISCUSSION TITLE
Cannot shutdown from gnome livecd

### DISCUSSION THREAD

**Discussion by CicadaSeventeen** [SpiralLinux/SpiralLinux-project#161] [Q&A]
![image](https://user-images.githubusercontent.com/86961676/195834462-7b0b6ea2-616e-4405-94ba-520fdd5d95d8.png)
Seems that this problem is on gnome only

**Comment by geckolinux** [comment]
Hi there, did this happen after installation with Calamares? Can you please confirm which release you used?
